src/ecmult_impl.h

```diff
@@ -200,9 +200,12 @@ static int secp256k1_ecmult_wnaf(int *wnaf, int len, const secp256k1_scalar *a,
         bit += now;
     }
 #ifdef VERIFY
-    CHECK(carry == 0);
-    while (bit < 256) {
-        CHECK(secp256k1_scalar_get_bits(&s, bit++, 1) == 0);
+    VERIFY_CHECK(carry == 0);
+
+    int verify_bit = bit;
+    while (verify_bit < 256) {
+        VERIFY_CHECK(secp256k1_scalar_get_bits(&s, verify_bit, 1) == 0);
+        verify_bit++;
```

Comment:
You need to wrap that piece of code in a new block , i.e., `{` and `}`. Then the compiler will accept it.

C90 has this strange rule that variable declarations must always come first in a block.